### PR TITLE
Add monitoring Surveys Report page

### PR DESCRIPTION
**Pull request by echaidemenos**
Adds monitoring Surveys Report page and format option for monitoring table

Screenshot from 2023-12-12 14-48-25

### CODE REVIEW

**Comment by ericboucher** [comment]
@Caesarh97 what's the most useful between number of surveys and number of pictures overall?

File: packages/api/src/monitoring/monitoring.service.ts [hunk 1]
```diff
@@ -206,4 +210,24 @@ export class MonitoringService {
...
+    return this.surveyRepository
+      .createQueryBuilder('survey')
+      .select('survey.site_id', 'siteId')
+      .addSelect('survey.id', 'surveyId')
+      .addSelect('survey.dive_date', 'diveDate')
+      .addSelect('survey.updated_at', 'updatedAt')
+      .addSelect('s.name', 'siteName')
+      .addSelect('u.email', 'userEmail')
+      .addSelect('u.full_name', 'userFullName')
+      .addSelect('COUNT(sm.id)::int', 'surveyMediaNum')
```
Comment (ericboucher) [inline packages/api/src/monitoring/monitoring.service.ts:224]: surveyMediaCount might be more accurate

**Comment by ericboucher** [comment]
@echaidemenos it might be useful to display the name of the table as a title at the top (above the table)

**Comment by Caesarh97** [comment]
> @Caesarh97 what's the most useful between number of surveys and number of pictures overall?

@ericboucher, Sorry, I didn’t see the comment. I think the number of surveys would be the most beneficial.

**Comment by echaidemenos** [comment]
> > @Caesarh97 what's the most useful between number of surveys and number of pictures overall?
> 
> @ericboucher, Sorry, I didn’t see the comment. I think the number of surveys would be the most beneficial.

I believe the number of surveys would make more sense in the sites page (to be added soon)
This table view shows a surveys for each line

**Pull request merged**